lib/runctl.js

```diff
@@ -190,6 +190,19 @@ function onRequest(req, callback) {
     }
     rsp = requestAllTargets(req, callback);
 
+  } else if (cmd === 'tracing') {
+    if (req.enabled) {
+      process.env.STRONGLOOP_TRACING = 1;
+    } else if (process.env.STRONGLOOP_TRACING !== null) {
+      delete process.env.STRONGLOOP_TRACING;
+    }
+
+    try {
```

Comment:
I can't find evidence that restart can throw. Did you see it throw? If so, that might be a bug.

cf. https://github.com/strongloop/strong-cluster-control/blob/master/lib/master.js#L281-L334